github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -140,6 +140,39 @@ periodics:
     - name: gcs
       secret:
         secretName: gcs
+- name: periodic-kubevirt-e2e-k8s-1.19-sig-network
+  annotations:
+    testgrid-create-test-group: "false"
+  interval: 8h
```

Comment:
Can you please include a `cron` entry instead of `interval` with specific execution times? like https://github.com/kubevirt/project-infra/blob/master/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml#L311

Reply:
updated to cron, but using each 8h, as was discussed offline, hope its ok ?